### PR TITLE
Dina review 1

### PR DESCRIPTION
**Pull request by dushnik**

### CODE REVIEW

File: Whisper_Audio.ipynb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (dushnik) [inline Whisper_Audio.ipynb:1]: 1. I suggest setting all variables in a single section:
audio_folder, transcription_folder, lang
2. You can change the names to audio_folder_path and transcription_folder_path.
3. You can add another variable for file suffix (.txt)

File: Whsiper_Video.ipynb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (dushnik) [inline Whsiper_Video.ipynb:1]: See audio file comments

**Review by dushnik — COMMENTED** [review]
I think you should change the dir structure:
Readme
resources -> img
colab -> audio + video

**Comment by Sourasky-DHLAB** [comment]
> I think you should change the dir structure: Readme resources -> img colab -> audio + video

**Comment by Sourasky-DHLAB** [comment]
fixed, thanks

**Pull request closed without merging**